src/components/Company/CompanyName/CorrectNameOptions.vue

```diff
@@ -67,23 +80,28 @@ export default class CorrectNameOptions extends Vue {
   // local properties
   private displayedOptions: Array<CorrectNameOptionIF> = []
   private panel = null as number
-  private submit = false
+  private submitId = ''
+  private currentPanelId = ''
   private isLoading = false
   private isFormValid = false
   private correctionNameOptions: Array<CorrectNameOptionIF> = [
     {
       id: CorrectionTypes.CORRECT_NAME,
-      description: 'Edit the company name',
-      component: ''// CorrectName
+      title: 'Edit the company name',
+      description: 'Correct typographical errors int he existing company name.',
+      component: CorrectCompanyName
     },
     {
       id: CorrectionTypes.CORRECT_NAME_TO_NUMBER,
```

Comment:
How is this handled? I see that the other two components name request correction and new NR alters the store state in those components itself. Shouldnt this option also alter the name request object in the store or is it handled some other way?

Reply:
This is the last Name Correction option that i'm currently working on in ticket #4587. It will do something very similar!